source/ncconvert/utility/Image.cpp

```diff
@@ -0,0 +1,168 @@
+#include "Image.h"
+#include "utility/EnumExtensions.h"
+
+#include "ncasset/Assets.h"
+#include "ncutility/NcError.h"
+
+#define STB_DXT_IMPLEMENTATION
+#define STB_IMAGE_IMPLEMENTATION
+#define STB_IMAGE_RESIZE_IMPLEMENTATION
+#include "stb/stb_dxt.h"
+#include "stb/stb_image.h"
+#include "stb/stb_image_resize2.h"
+
+#include <utility>
+
+namespace
+{
+auto ToVector(const nc::convert::RawImage& image, uint32_t numBytes) -> std::vector<unsigned char>
+{
+    auto out = std::vector<unsigned char>(numBytes);
+    std::memcpy(out.data(), image.get(), numBytes);
+    return out;
+}
+
+auto GetPaddedDimension(int dimension) -> int
+{
+    return (dimension + 3) & ~3;
+}
+} // anonymous namespace
+
+namespace nc::convert
+{
+auto MakeRawImage(int width,
+                  int height,
+                  const unsigned char fillRGBA[4]) -> RawImage
+{
+    const auto numBytes = width * height * 4;
+    auto bytes = (unsigned char*)std::malloc(numBytes);
+    for (auto i = 0; i < numBytes; i += 4)
+    {
+        std::memcpy(bytes + i, fillRGBA, 4);
```

Comment:
Am I understanding this correctly - does this always set every texture's Alpha value to 1.0 (aka 255?)